runtime/builtin_types.go

```diff
@@ -193,6 +193,81 @@ func builtinAbs(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	return Abs(f, args[0])
 }
 
+func zipLongestImpl(f *Frame, args Args) ([]*Object, *BaseException) {
```

Comment:
Call this zipLongest since it's slightly simpler. Also, please move it to the end of the file. I try to group all the helper functions at the end of the file (although I'm sure I've missed some).